app/src/main/kotlin/com/ayatk/biblio/domain/usecase/DetailUseCaseImpl.kt

```diff
@@ -31,25 +31,27 @@ import io.reactivex.Single
 import javax.inject.Inject
 
 class DetailUseCaseImpl @Inject constructor(
-    private val novelRepository: NovelRepository,
-    private val indexRepository: IndexRepository,
-    private val schedulerProvider: SchedulerProvider
+  private val novelRepository: NovelRepository,
+  private val indexRepository: IndexRepository,
+  private val schedulerProvider: SchedulerProvider
 ) : DetailUseCase {
 
   override fun getLibrary(novel: Novel): Single<Library> =
-      // TODO とりあえず今はタグ実装してないのでそのまま変換して流してる
-      Library("remote", novel, emptyList()).toSingle()
+  // TODO とりあえず今はタグ実装してないのでそのまま変換して流してる
+    Library("remote", novel, emptyList()).toSingle()
 
   override fun saveLibrary(library: Library): Completable {
-    TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
+    TODO("not implemented")
```

Comment:
**[detekt]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the NotImplementedDeclaration should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.